src/main/entities/NewPassword.java

```diff
@@ -0,0 +1,146 @@
+package entities;
+
+public class NewPassword extends Password {
+
+    private int passlen;
```

Comment:
Please spell out instance variables. The wider the scope, the more clear the name needs to be.